### PR TITLE
start fleshing out migration guide

### PR DESCRIPTION
**Pull request by Rich-Harris**

### CODE REVIEW

File: documentation/migrating/02-pkg.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: package.json
+---
+
+### dependencies
+
+Remove `polka` or `express`, if you're using one of those, and any middleware such as `sirv` or `compression`.
+
+### devDependencies
+
+Remove `sapper` from your `devDependencies` and replace it with `@sveltejs/kit`, `@vite`, and whichever [adapter](/docs#adapters) you plan to use (see [next section](#project-files-configuration)).
```
Comment (benmccann) [inline documentation/migrating/02-pkg.md:11]: I think it's just `vite` instead of `@vite`

**Review by benmccann — COMMENTED** [review]
lgtm

File: documentation/migrating/02-pkg.md (hunk 1 above)
Comment (Rich-Harris) [reply documentation/migrating/02-pkg.md]: good catch, fixed

File: documentation/migrating/01-migrating.md [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
+title: Migrating from Sapper
 ---
 
-# Migrating from Sapper
+SvelteKit is the successor to Sapper and shares many elements of its design.
 
-SvelteKit is largely backwards-compatible with Sapper. However, there are a number of differences. You may find it helpful to view the [examples](https://github.com/sveltejs/kit/tree/master/examples) while migrating.
+If you have an existing Sapper app that you plan to migrate to SvelteKit, however, there are a number of changes you will need to make. You may find it helpful to view the [examples](https://github.com/sveltejs/kit/tree/master/examples) while migrating.
 
-- Remove `rollup.config.js` or `webpack.config.js` and replace with [`vite.config.js`](https://github.com/sveltejs/kit/blob/master/packages/create-svelte/template/vite.config.js)
```
Comment (antony) [inline documentation/migrating/01-migrating.md:9]: I sorta feel the shortlist is a useful starter for ten...

**Pull request merged**